docs/implementations/python/api.rst

```diff
@@ -7,6 +7,8 @@ Python API
     :imported-members:
 
 
+    .. manually documented as parse_annotation is undocumented
```

Comment:
I'll add a cross-reference to the manual documentation here:
```rst
:meth:`AnnotationStringParser.__call__`
```

The `autodoc` sees that it is a function and pulls the signature from `AnnotationStringParser.__call__`, but nothing else.